### PR TITLE
Update/supporters images

### PR DESCRIPTION
**Pull request by losndu**
## Summary by CodeRabbit

- **New Features**
	- Added new SVG icon components for "Abcde," "Megaeth," "ModularMedia," and "Spire" to enhance visual representation.
	- Updated existing SVG components "EigenLayer" and "MegaEth" to allow automatic height adjustment for better responsiveness.
	- Introduced a new fill property in "MegaEth" for dynamic color customization of SVG paths.

- **Bug Fixes**
	- Adjusted layout and styling for better visual presentation of lists and images, including additional left padding for improved spacing.

- **Documentation**
	- Updated text content to reflect a shift in focus to technical analyses of Ethereum infrastructure.

### CODE REVIEW

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -8,16 +8,72 @@
...
+        {
+            imageSrc: "/shun-kanoki.webp",
+            alt: "shun-kanoki",
+            link: "https://x.com/shunkakinoki",
+        },
+        {
+            imageSrc: "/lightso.webp",
+            alt: "lightso",
+            link: "https://light.so/home",
+        },
```
Comment (iankressin) [inline src/routes/+page.svelte]: DefiDave's image redirects to shunkakinoki's twitter, Fuel Labs redirects to DefiDave's and Shun Kakinoki redirect to Fuel Labs website.

To be honest, this is not the type of quality that I expect from a PR. It  doesn't meet the minimal requirements specified in the [task](https://www.notion.so/Update-supporters-images-10a28d0d3291802fb967d838bd713cef?pvs=4)

Please, review if all the images redirect to the correct links.

**Review by iankressin — CHANGES_REQUESTED** [review]

**Review by iankressin — CHANGES_REQUESTED** [review]
You need to make sure that that link icon show up when there's an image with a darker background. You can do that by including `darkBackground` prop to the list of images, and them, rendering the icon white, if the background is dark.

image


image

image

image

image

File: static/lightso.webp
Comment (iankressin) [inline static/lightso.webp:1]: Use this svg for Light.so

```html
<svg width="171" height="171" viewBox="0 0 171 171" fill="none" xmlns="http://www.w3.org/2000/svg" class="m-2.5 size-8 fill-text"><title>Light</title><path fill-rule="evenodd" clip-rule="evenodd" d="M85.3339 6.1218e-05C70.2013 -0.0181096 55.3391 4.00917 42.2863 11.6648H128.376C115.325 4.01021 100.464 -0.0170325 85.3339 6.1218e-05ZM147.111 26.5013C150.555 30.1093 153.677 34.0114 156.442 38.163H14.2189C16.2609 35.1417 18.4958 32.2552 20.9096 29.5217H20.8069C21.0161 29.2832 21.2385 29.0605 21.4603 28.8385L21.4604 28.8385C21.6486 28.65 21.8364 28.4621 22.0153 28.2652C22.2359 28.0204 22.4489 27.7686 22.6619 27.5168L22.6619 27.5168L22.6619 27.5168L22.6619 27.5168L22.662 27.5168C22.9535 27.1722 23.245 26.8277 23.5557 26.5013H23.7008C26.6233 23.3899 29.7735 20.5005 33.1253 17.857H80.3083C76.9552 20.4989 73.8049 23.3885 70.8838 26.5013H147.111ZM168.134 64.6555C167.145 60.6805 165.867 56.7829 164.31 52.9938H53.587C54.7808 50.0391 56.1382 47.1532 57.6529 44.3494H10.4698C8.95492 47.1526 7.5986 50.0386 6.4071 52.9938H6.35566C6.27298 53.1973 6.19952 53.4067 6.12601 53.6162C6.05987 53.8046 5.9937 53.9932 5.92074 54.1778C5.82779 54.4101 5.72859 54.6424 5.62879 54.8762C5.46911 55.2501 5.30793 55.6276 5.16851 56.0142H5.24411C4.18204 58.8433 3.27442 61.7279 2.52542 64.6555H168.134ZM170.448 79.4963H47.455C47.6324 76.6012 47.957 73.717 48.4275 70.8549H1.24762C0.774551 73.7168 0.448901 76.601 0.271943 79.4963H0.220504C0.196255 79.8426 0.186893 80.189 0.177624 80.532C0.170728 80.7871 0.163883 81.0404 0.150997 81.2904C0.14239 81.4854 0.126986 81.6785 0.111563 81.8717C0.0945112 82.0854 0.0774374 82.2994 0.0695067 82.5166H0.105827C0.0941012 82.8195 0.0788872 83.1215 0.0636691 83.4235L0.0636631 83.4236C0.0318398 84.0552 0 84.6871 0 85.3286C0 87.2889 0.0906146 89.2309 0.220504 91.158H170.448C170.578 89.2309 170.669 87.2889 170.669 85.3286C170.669 83.3684 170.578 81.4263 170.448 79.4963ZM30.1045 150.349H77.2875C81.0978 153.586 85.1844 156.482 89.5002 159.006H128.382C115.329 166.661 100.467 170.689 85.3346 170.67C72.3833 170.676 59.6026 167.717 47.9718 162.02H47.8932C47.8096 161.976 47.7284 161.931 47.6477 161.886L47.6476 161.885L47.6474 161.885L47.6473 161.885C47.5444 161.828 47.4424 161.77 47.3375 161.718C47.0836 161.592 46.8342 161.457 46.5845 161.322L46.5842 161.322L46.5838 161.322C46.4208 161.234 46.2576 161.145 46.0931 161.06C44.9029 160.443 43.7217 159.815 42.5648 159.145C42.5164 159.116 42.4662 159.089 42.4156 159.062C42.3716 159.039 42.3274 159.016 42.2838 158.991H42.3172C38.0019 156.472 33.9154 153.581 30.1045 150.349ZM56.3811 123.854H9.19802C10.7338 126.832 12.4425 129.717 14.3151 132.496H14.2244C14.4105 132.774 14.6115 133.039 14.8124 133.304C14.9735 133.517 15.1345 133.729 15.2877 133.948C15.4406 134.167 15.5918 134.401 15.7449 134.637L15.7454 134.638C15.9369 134.934 16.1313 135.234 16.336 135.516H16.4326C18.6175 138.548 20.9967 141.436 23.5553 144.16H147.101C150.545 140.551 153.667 136.648 156.432 132.496H61.4892C59.6174 129.718 57.9116 126.833 56.3811 123.854ZM0.858082 97.3539H48.0411C48.4682 100.263 49.049 103.148 49.781 105.995H168.134C167.145 109.97 165.867 113.868 164.31 117.657H6.35575C5.21845 114.831 4.23334 111.947 3.40457 109.016H3.35627C3.28912 108.785 3.23349 108.549 3.17785 108.312C3.13087 108.113 3.08387 107.913 3.02994 107.717C2.96283 107.47 2.88911 107.227 2.81537 106.983C2.7165 106.656 2.61759 106.329 2.53454 105.995H2.59796C1.86736 103.147 1.28663 100.263 0.858082 97.3539Z" fill="currentColor"></path></svg>
```

File: static/azuki.webp
Comment (iankressin) [inline static/azuki.webp:1]: Use this svg for Azuki:

```html
<svg xmlns="http://www.w3.org/2000/svg" width="125" height="24" viewBox="0 0 125 24" fill="none">
<path d="M11.381 24C11.6167 24 11.7177 23.8683 11.7851 23.6708L13.9401 17.251L16.0951 23.6708C16.1625 23.8683 16.2635 24 16.4992 24H27.5099C27.7793 24 27.914 23.8354 27.813 23.572L19.6307 0.197532C19.597 0.0658447 19.4623 0 19.3276 0H8.61992C8.45156 0 8.35054 0.0658447 8.2832 0.197532L0.0335394 23.572C-0.0674768 23.8354 0.0672084 24 0.336585 24H11.381ZM53.402 24C53.57 24 53.7043 23.8683 53.7043 23.7037V13.9259C53.7043 13.7613 53.57 13.6296 53.402 13.6296H47.7927C47.7592 13.6296 47.6584 13.6296 47.6248 13.5638C47.5912 13.4979 47.6248 13.4321 47.6584 13.3663L53.6371 0.625514C53.8387 0.197531 53.6036 0 53.3348 0H29.8901C29.7221 0 29.5878 0.164607 29.5878 0.329216V10.107C29.5878 10.2716 29.7221 10.4033 29.8901 10.4033H35.5665C35.6001 10.4033 35.7009 10.4033 35.7345 10.4691C35.768 10.535 35.7345 10.6008 35.7009 10.6667L29.6549 23.4074C29.4534 23.8025 29.6885 24 29.9572 24H53.402ZM68.1931 24C75.0327 24 80.6931 18.6996 80.6931 12.0823V0.329216C80.6931 0.131685 80.5583 0 80.3561 0H69.3723C69.2039 0 69.0691 0.131685 69.0691 0.329216V11.8848C69.0691 12.3786 68.6648 12.7407 68.1931 12.7407C67.7214 12.7407 67.3171 12.3786 67.3171 11.8848V0.329216C67.3171 0.131685 67.1823 0 67.0138 0H56.03C55.8278 0 55.6931 0.131685 55.6931 0.329216V12.0823C55.6931 18.6996 61.3535 24 68.1931 24ZM94.7037 24C94.9051 24 95.0394 23.8683 95.0394 23.6708V15.8025H95.9791L99.3355 23.6708C99.4362 23.8683 99.5033 24 99.7718 24H111.15C111.385 24 111.452 23.9012 111.318 23.6049L106.35 12.0494L111.318 0.39506C111.452 0.098764 111.385 0 111.15 0H99.8054C99.5369 0 99.4698 0.131685 99.3691 0.329216L95.912 8.42799H95.0394V0.329216C95.0394 0.164607 94.9051 0 94.7037 0H83.6277C83.4263 0 83.2921 0.0987636 83.2921 0.329216V23.6708C83.2921 23.8683 83.4263 24 83.6277 24H94.7037ZM124.664 24C124.832 24 125 23.8683 125 23.6708V0.329035C125 0.164424 124.832 0 124.664 0H113.578C113.411 0 113.243 0.164424 113.243 0.329035V23.6708C113.243 23.8683 113.411 24 113.578 24H124.664Z" fill="#C03540"/>
</svg>
```

File: static/fuel-labs.webp
Comment (iankressin) [inline static/fuel-labs.webp:1]: Use this for Fuel

```svg
<svg xmlns="http://www.w3.org/2000/svg" width="354" height="500" viewBox="0 0 354 500" fill="none">
<g clip-path="url(#clip0_1_15040)">
<path d="M88.3564 491.819V388.481H102.685V485.662H158.745L158.426 388.583L173.074 388.481V491.819C173.074 493.989 172.212 496.07 170.678 497.604C169.144 499.138 167.063 500 164.893 500H96.537C95.4627 500 94.399 499.788 93.4064 499.377C92.4139 498.966 91.5121 498.364 90.7525 497.604C89.9929 496.844 89.3903 495.942 88.9792 494.95C88.568 493.957 88.3564 492.894 88.3564 491.819Z" fill="black"/>
<path d="M281.125 491.819V388.481H295.477V485.662H353.264V500.014H289.306C288.23 500.014 287.165 499.802 286.172 499.39C285.178 498.978 284.276 498.374 283.516 497.613C282.756 496.852 282.154 495.948 281.744 494.954C281.333 493.96 281.123 492.895 281.125 491.819Z" fill="black"/>
<path d="M0 500H14.3287V451.569H68.3241V437.218H14.3287V402.805H67.4398C68.2125 402.808 68.9701 402.592 69.6254 402.182C70.2806 401.773 70.8069 401.186 71.1435 400.491L76.8472 388.486H8.18056C7.10649 388.485 6.04283 388.697 5.05034 389.107C4.05786 389.518 3.15601 390.12 2.39632 390.879C1.63662 391.638 1.03397 392.54 0.622799 393.532C0.211631 394.524 -1.72195e-07 395.588 0 396.662V500Z" fill="black"/>
<path d="M188.676 396.657C188.676 395.583 188.888 394.52 189.299 393.528C189.71 392.535 190.313 391.634 191.072 390.874C191.832 390.115 192.734 389.513 193.726 389.102C194.719 388.692 195.783 388.481 196.857 388.481H265.519L259.81 400.486C259.474 401.182 258.948 401.769 258.293 402.178C257.637 402.588 256.879 402.804 256.107 402.801H203.005V437.213H257V451.565H203.005V485.657H265.505V500.009H196.857C194.687 500.009 192.606 499.147 191.072 497.613C189.538 496.079 188.676 493.998 188.676 491.829V396.657Z" fill="black"/>
<path d="M23.4861 4.56429e-07C20.4021 -0.000607592 17.3481 0.606316 14.4987 1.7861C11.6493 2.96588 9.06015 4.69542 6.87921 6.87594C4.69826 9.05645 2.9682 11.6452 1.78786 14.4944C0.607512 17.3436 -5.99417e-08 20.3975 0 23.4815V353.167H292.268C297.169 353.17 302.022 352.208 306.55 350.334C311.079 348.46 315.193 345.711 318.657 342.245L342.338 318.574C349.329 311.581 353.258 302.097 353.259 292.208V4.56429e-07H23.4861Z" fill="#00F58C"/>
<path d="M57.2546 45.4074H230.634L115.759 160.259C112.907 163.108 109.04 164.708 105.009 164.708C102.13 164.708 99.309 163.891 96.8757 162.351C94.4424 160.811 92.4963 158.612 91.2638 156.009L46.7592 61.9352C45.9225 60.1684 45.5469 58.2185 45.6676 56.2673C45.7882 54.3161 46.4011 52.4273 47.449 50.777C48.497 49.1267 49.9458 47.7687 51.6605 46.8298C53.3751 45.8908 55.2997 45.4015 57.2546 45.4074Z" fill="black"/>
<path d="M45.4167 307.741V196.005C45.418 193.142 46.556 190.397 48.5805 188.374C50.6051 186.35 53.3505 185.213 56.213 185.213H167.986L45.4167 307.741Z" fill="black"/>
<path d="M196.477 156.731C191.365 161.839 184.434 164.708 177.208 164.708H140.208L251.555 53.3842C256.667 48.2768 263.598 45.4077 270.824 45.4073H307.824L196.477 156.731Z" fill="black"/>
</g>
<defs>
<clipPath id="clip0_1_15040">
<rect width="353.241" height="500" fill="white"/>
</clipPath>
</defs>
</svg>
```

File: static/starknet.webp
Comment (iankressin) [inline static/starknet.webp:1]: Use this for Starknet:

```svg
<svg xmlns="http://www.w3.org/2000/svg" width="158" height="158" viewBox="0 0 158 158" fill="none">
<path fill-rule="evenodd" clip-rule="evenodd" d="M-3.68439e-06 78.9988C-3.68439e-06 122.629 35.3685 157.998 78.9988 157.998C122.629 157.998 158 122.629 158 78.9988C158 35.3685 122.629 0 78.9988 0C35.3685 0 -3.68439e-06 35.3685 -3.68439e-06 78.9988Z" fill="#0C0C4F"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M44.1542 60.3876L46.1043 54.3628C46.5007 53.1374 47.4678 52.1839 48.6977 51.8079L54.752 49.9461C55.59 49.6901 55.5968 48.5078 54.7656 48.2383L48.7385 46.2881C47.5154 45.8918 46.5618 44.9246 46.1836 43.6947L44.324 37.6405C44.0681 36.8047 42.8858 36.7956 42.6162 37.6291L40.6661 43.654C40.2697 44.877 39.3026 45.8306 38.0727 46.2089L32.0184 48.0684C31.1804 48.3266 31.1713 49.5067 32.0048 49.7762L38.0319 51.7263C39.255 52.1227 40.2086 53.0921 40.5868 54.322L42.4464 60.374C42.7023 61.212 43.8846 61.2211 44.1542 60.3876Z" fill="#FAFAFA"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M139.848 56.881C137.352 54.09 133.457 52.5186 129.67 51.8742C125.853 51.2553 121.85 51.3127 118.082 51.9822C110.457 53.272 103.53 56.4292 97.4881 60.3241C94.3506 62.2356 91.675 64.4474 88.903 66.7C87.5674 67.8392 86.3497 69.0524 85.0815 70.2482L81.6163 73.696C77.851 77.6317 74.1398 81.2165 70.549 84.1878C66.9437 87.1454 63.573 89.3917 60.2512 90.9604C56.9315 92.5372 53.3794 93.4645 48.7495 93.6129C44.1603 93.7748 38.7307 92.9465 32.9229 91.5794C27.0839 90.2183 20.9523 88.2782 14.1004 86.6088C16.4912 93.2414 20.0914 99.1027 24.7137 104.461C29.3902 109.726 35.2297 114.524 42.7314 117.68C50.1247 120.906 59.4157 122.064 68.1031 120.317C76.8135 118.64 84.4573 114.61 90.6359 109.948C96.8304 105.239 101.842 99.8748 106.067 94.298C107.233 92.7572 107.85 91.8949 108.694 90.6906L111.027 87.2351C112.648 85.0977 114.124 82.6641 115.728 80.5464C118.874 76.1112 121.976 71.6811 125.58 67.5997C127.394 65.5296 129.307 63.5495 131.565 61.6466C132.691 60.7178 133.908 59.8088 135.256 58.99C136.624 58.1068 138.069 57.4155 139.848 56.881Z" fill="#EC796B"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M139.848 56.8808C137.167 50.1148 132.182 44.4194 125.491 40.2167C118.841 36.0601 109.605 33.9389 100.452 35.7471C95.9307 36.6216 91.5501 38.3052 87.7059 40.5472C83.8791 42.7803 80.4499 45.4685 77.4812 48.3524C75.9992 49.7992 74.654 51.3084 73.3176 52.8262L69.8538 57.2423L64.5039 64.3512C57.6836 73.4976 50.3391 84.2166 38.2863 87.3927C26.454 90.5108 21.3221 87.7493 14.1004 86.6085C15.4209 90.0176 17.0565 93.3284 19.2739 96.2409C21.45 99.2124 24.0204 102.003 27.216 104.396C28.8309 105.545 30.5359 106.679 32.4295 107.641C34.3146 108.57 36.3474 109.388 38.5172 110.003C42.8332 111.186 47.6923 111.601 52.397 110.964C57.1039 110.336 61.6029 108.845 65.5355 106.864C69.4971 104.901 72.9481 102.511 76.0296 99.9891C82.1551 94.9021 86.9197 89.2815 90.9444 83.6C92.9688 80.7594 94.8064 77.8652 96.5059 74.9702L98.5062 71.5236C99.1177 70.5161 99.7362 69.5024 100.364 68.5584C102.899 64.7654 105.378 61.7241 108.389 59.4413C111.358 57.0995 115.493 55.3693 121.018 54.9674C126.52 54.5606 132.872 55.312 139.848 56.8808Z" fill="#FAFAFA"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M110.081 113.097C110.081 118.067 114.112 122.097 119.081 122.097C124.051 122.097 128.076 118.067 128.076 113.097C128.076 108.128 124.051 104.097 119.081 104.097C114.112 104.097 110.081 108.128 110.081 113.097Z" fill="#EC796B"/>
</svg>
```

File: static/metalex.webp
Comment (iankressin) [inline static/metalex.webp:1]: Use this for Matalex:

```svg
<svg xmlns="http://www.w3.org/2000/svg" width="127" height="127" viewBox="0 0 127 127" fill="none">
<circle cx="63.5" cy="63.5" r="63.5" fill="#DAFF00"/>
<path d="M98.7778 41.4479V54.6724C98.7778 55.64 97.9897 56.4244 97.0175 56.4244H83.9884C83.0162 56.4244 82.2281 55.64 82.2281 54.6724V50.7161C82.2281 50.133 82.5195 49.5882 83.0054 49.2626L96.4835 40.2323C97.4614 39.5772 98.7778 40.2746 98.7778 41.4479Z" fill="#191A18"/>
<path d="M93.4277 23.6845C93.4277 22.5945 92.54 21.7109 91.4449 21.7109H91.2356C90.8474 21.7109 90.4677 21.8244 90.1437 22.0372L57.1094 43.7301C56.5532 44.0953 56.2185 44.7142 56.2185 45.3775V54.4724C56.2185 55.5624 57.1063 56.446 58.2014 56.446H62.981C63.3682 56.446 63.7469 56.3332 64.0704 56.1215L92.5342 37.4943C93.0919 37.1293 93.4277 36.5096 93.4277 35.8452V23.6845Z" fill="#191A18"/>
<path d="M63.2605 23.6845C63.2605 22.5945 62.3727 21.7109 61.2776 21.7109H61.0684C60.6802 21.7109 60.3005 21.8244 59.9764 22.0372L26.9422 43.7301C26.386 44.0953 26.0513 44.7142 26.0513 45.3775V54.4724C26.0513 55.5624 26.939 56.446 28.0341 56.446H32.8138C33.201 56.446 33.5796 56.3332 33.9031 56.1215L62.367 37.4943C62.9247 37.1293 63.2605 36.5096 63.2605 35.8452V23.6845Z" fill="#191A18"/>
<path d="M107.235 75.0341C107.226 75.0293 107.216 75.0267 107.206 75.0265C107.196 75.0263 107.186 75.0286 107.177 75.0332C107.168 75.0377 107.16 75.0444 107.154 75.0526C107.148 75.0609 107.144 75.0704 107.143 75.0805C106.675 78.2917 105.915 81.4229 104.862 84.4743C102.117 92.4327 97.4118 99.0638 90.7478 104.368C88.1939 106.399 85.517 108.118 82.7171 109.525C78.6634 111.56 74.3671 112.895 69.8284 113.53C66.0217 114.063 62.1658 114.116 58.2607 113.689C49.9302 112.78 41.9764 109.296 35.4897 104.012C34.6624 103.338 33.9017 102.666 33.2076 101.996C25.5983 94.6475 21.0883 85.7201 19.6776 75.2135C19.6745 75.191 19.6661 75.1697 19.6529 75.1513C19.6397 75.133 19.6222 75.1182 19.602 75.1082C19.5818 75.0983 19.5594 75.0935 19.5369 75.0943C19.5144 75.0952 19.4925 75.1016 19.473 75.1129C18.9122 75.4459 18.3478 75.7155 17.7799 75.9217C16.4588 76.401 15.1455 75.447 15.1978 74.0383C15.2286 73.1986 15.7806 72.6296 16.4604 72.2585C19.4377 70.6379 22.8009 70.2854 26.1258 70.74C27.4314 70.9178 28.4064 71.9739 27.9927 73.327C27.7036 74.2748 26.7378 74.7743 25.7844 74.6722C25.0852 74.597 24.3706 74.5109 23.6406 74.414C23.6245 74.4117 23.6081 74.4132 23.5927 74.4183C23.5773 74.4235 23.5632 74.4322 23.5517 74.4437C23.5403 74.4553 23.5316 74.4694 23.5265 74.4849C23.5213 74.5004 23.5199 74.5169 23.5222 74.5331C23.9097 77.4154 24.5731 80.2596 25.5122 83.0656C28.2588 91.2673 33.1907 97.801 40.3078 102.667C45.0275 105.891 50.2408 108.314 55.8386 109.514C61.5409 110.735 67.5524 110.53 73.2317 108.898C78.4224 107.408 83.1846 105.087 87.5183 101.935C91.6387 98.9406 94.9733 95.3341 97.522 91.1157C99.7827 87.3757 101.439 83.1868 102.491 78.5489C102.777 77.2902 103.03 75.9619 103.251 74.564C103.253 74.5492 103.251 74.5341 103.247 74.5199C103.242 74.5056 103.235 74.4926 103.225 74.4817C103.214 74.4709 103.202 74.4625 103.188 74.4572C103.174 74.4519 103.159 74.4498 103.144 74.4511C102.583 74.5027 102.018 74.5656 101.451 74.6398C100.979 74.7006 100.584 74.6826 100.269 74.5857C99.1552 74.2439 98.5801 73.0564 99.0399 71.9708C99.3767 71.1714 100.033 70.859 100.896 70.7446C104.438 70.273 107.59 70.6364 110.598 72.3249C111.859 73.0332 112.365 74.8037 110.951 75.6959C110.592 75.9217 110.136 76.0005 109.583 75.9325C108.636 75.8165 108.026 75.4671 107.235 75.0341Z" fill="#191A18"/>
</svg>
```

**Review by iankressin — COMMENTED** [review]
We're going to need to update a few logos. Most of the companies' logos had a really poor quality

File: src/routes/+page.svelte (hunk 1 above)
Comment (2077collective) [reply src/routes/+page.svelte:76]: I've double-checked the links and confirmed that the images redirect to the correct URLs as specified in the PR. It's possible that there might have been a temporary misdirection or caching issue.

Could you please reverify the links or provide more details about the incorrect redirects you're experiencing.

The screenshot you also shared above shows all images with the right corresponding links

Comment (losndu) [reply src/routes/+page.svelte]: I've double-checked the links and confirmed that the images redirect to the correct URLs as specified in the PR. It's possible that there might have been a temporary misdirection or caching issue.

Could you please reverify the links or provide more details about the incorrect redirects you're experiencing.

The screenshot you also shared above shows all images with the right corresponding links

**Review by iankressin — APPROVED** [review]

**Pull request merged**